docs/3.0/build.md

```diff
@@ -0,0 +1,247 @@
+---
+---
+# Building a Tabris.js App
+
+Tabris.js utilizes [Apache Cordova](http://cordova.apache.org) to build and package apps. Apps can be built without any local setup [using the free online build service](#build-service) on tabrisjs.com. To [build an app on your local machine](#local-build), you need to setup developer tools like Xcode, Visual Studio or the Android SDK. The following features are supported by the two different build types.
+
+|                           | Build Service | Local Build |
+| :------------------------ |:---------------:| :---------------: |
+| Building iOS Apps         |       ✓         |       ✓      |
+| Building Android Apps     |       ✓         |       ✓      |
+| [Integrate Cordova Plugins](cordova.md)     |       ✓      |       ✓      |
+| [Cordova Build Hooks](http://cordova.apache.org/docs/en/edge/guide_appdev_hooks_index.md.html#Hooks%20Guide)       |       ✓      |       ✓      |
+| Custom Project Structure  |       ✓      |       ✓      |
+| Own Build Scripts         |              |       ✓      |
+| Using own build hardware  |              |       ✓      |
+| Other SCMs than Git       |              |       ✓      |
+
+> :point_right: The online build service is free for unlimited public GitHub repositories and 1 private repository. To build from unlimited private repositories, you need a [Pro account](https://tabrisjs.com/pricing/). [Local builds](#local-build) are free for everyone.
+
+## Project Layout
+
+To prepare your project for the build, you have to create a subdirectory named `cordova` that contains the build configuration. The layout of a Tabris.js project might look like this:
+```
+/
+|- cordova/
+    |- config.xml
+|- src/
+    |- <.js files>
+|- test/
+    |- <.spec.js files>
+|- package.json
+|- .tabrisignore
+```
+
+### The package.json file
+
+See [package.json | npm documentation](https://docs.npmjs.com/files/package.json) for information about the package.json format.
+
+`package.json` contains, among other configurations, references to the app's main script and npm module dependencies:
+
+```json
+{
+  // ...
+  "main": "src/app.js",
+  "dependencies": {
+    "tabris": "^3.0.0",
+    "left-pad": "^1.1.3",
+    // ...
+  }
+  // ...
+}
+```
+
+Dependencies are automatically installed during the build process.
+
+#### Build scripts
+
+When a Tabris.js app is built, `build` scripts given in the `package.json` are executed before the JavaScript code is bundled into the app. They can be used to transpile the JavaScript app code.
```

Comment:
"transpile"? "compile"? "cross-compile"?

Reply:
I think "compile" implies that bytecode would be the final output.  "cross-compile" is not a term I've heard before.